### DISCUSSION TITLE
Change or remove left pane minimum width

### DISCUSSION THREAD

**Discussion by toazd** [ezra-bible-app/ezra-bible-app#1100] [General]
If I grab the divider between the left and right panes I can resize the left pane all the way to the far right (the right pane pretty much disappears) as in the following screenshot:

[https://imgur.com/TAxZGLR](https://imgur.com/TAxZGLR)

But if I try to resize the panes so that the right size is larger the left pane has a minimum width that prevents the same behavior as in the following screenshot (the right size is as large as is allowed):

[https://imgur.com/0kQRThb](https://imgur.com/0kQRThb)

Is it possible to remove the minimum width for the left pane so that the right side can be made larger?

**Comment by tobias-klein** [comment]
The reason for this minimum width is the size of the content of the tool panel. When reducing the width it does not work anymore at some point with the content and that's why we have the minimum width. If you want to use the full width of the screen for the Bible browser you can hide the tool panel by clicking on the currently active icon of the respective tool panel.
